packages/labs/gen-manifest/package.json

```diff
@@ -39,10 +39,11 @@
         "build"
       ],
       "files": [
-        "goldens/**/*"
+        "goldens/**/*",
+        "../test-projects/test-element-a"
```

Comment:
Note, the manifest above includes a bunch of new files that were added in #3384. I believe CI didn't flag the goldens as out of date because we weren't including the input files in the `wireit` config, so the github action used the cached result. This should prevent that going forward.